src/containers/ui/text-fields.svelte

```diff
@@ -71,9 +68,9 @@
         isOutline
         isWithItem
         label="Telegram"
-        pattern="[A-Za-z0-9_](5, 32)"
+        pattern="[A-Za-z0-9_]{'{'}5,32{'}'}"
```

Comment:
using `minlength` and `maxlength` would probably look better

Reply:
Need to check it out